spec/abstractops.html

```diff
@@ -462,12 +485,14 @@ <h1>ToRelativeTemporalObject ( _options_ )</h1>
           1. Return ! CreateTemporalDate(_value_.[[ISOYear]], _value_.[[ISOMonth]], _value_.[[ISODay]], _value_.[[Calendar]]).
         1. Let _calendar_ be ? GetTemporalCalendarWithISODefault(_value_).
         1. Let _fieldNames_ be ? CalendarFields(_calendar_, « *"day"*, *"hour"*, *"microsecond"*, *"millisecond"*, *"minute"*, *"month"*, *"monthCode"*, *"nanosecond"*, *"second"*, *"year"* »).
+        1. Append *"timeZone"* to _fieldNames_.
+        1. Append *"offset"* to _fieldNames_.
         1. Let _fields_ be ? PrepareTemporalFields(_value_, _fieldNames_, «»).
+        1. Let _offsetString_ be ! Get(_fields_, *"offset"*).
+        1. Let _timeZone_ be ! Get(_fields_, *"timeZone"*).
         1. Let _dateOptions_ be OrdinaryObjectCreate(*null*).
         1. Perform ! CreateDataPropertyOrThrow(_dateOptions_, *"overflow"*, *"constrain"*).
         1. Let _result_ be ? InterpretTemporalDateTimeFields(_calendar_, _fields_, _dateOptions_).
```

Comment:
We can keep a consistent (_result_, _offsetString_, _timeZone_) definition order.
```suggestion
        1. Let _dateOptions_ be OrdinaryObjectCreate(*null*).
        1. Perform ! CreateDataPropertyOrThrow(_dateOptions_, *"overflow"*, *"constrain"*).
        1. Let _result_ be ? InterpretTemporalDateTimeFields(_calendar_, _fields_, _dateOptions_).
        1. Let _offsetString_ be ! Get(_fields_, *"offset"*).
        1. Let _timeZone_ be ! Get(_fields_, *"timeZone"*).
```